questionnaire/questionnaire/utils/taskRunner/index.js

```diff
@@ -0,0 +1,139 @@
+'use strict';
+
+function getDataRef(ref, data) {
+    if (typeof ref === 'string' && ref[0] === '$') {
+        let objValue = data;
+
+        for (let i = 1, keys = ref.split('.'), len = keys.length; i < len; i += 1) {
+            objValue = objValue[keys[i]];
+        }
+
+        return objValue;
+    }
+
+    return ref;
+}
+
+async function wait(milliseconds) {
+    return new Promise(resolve => {
+        setTimeout(resolve, milliseconds);
+    });
+}
+
+function mutateObjectValues({key, value, valueTransformers, isMutableObject, ignoreValue}) {
+    if (ignoreValue && ignoreValue(value) === true) {
+        return value;
+    }
+
+    if (isMutableObject(value)) {
+        Object.entries(value).forEach(([k, v]) => {
+            value[k] = mutateObjectValues({
+                key: k,
```

Comment:
Can we "take advantage" of object destructing here and just name `k`, and `v` to `key` and `value`.

Reply:
There are "key" / "value" parameters at the top level